conftest.py

```diff
@@ -4,5 +4,8 @@
 
 
 @pytest.fixture
-def jp_server_config(jp_server_config):
-    return {"ServerApp": {"jpserver_extensions": {"jupyterlab_git": True}}}
+def jp_server_config(jp_server_config, jp_root_dir):
+    return {
+        "ServerApp": {"jpserver_extensions": {"jupyterlab_git": True}},
+        "JupyterLabGit": {"excluded_paths": [f"/test-path/*"]},
```

Comment:
Unneeded f-string and could we use more precise rule so we avoid hard-to-find bug if somebody uses `test-path` in tests :wink: 

```suggestion
        "JupyterLabGit": {"excluded_paths": ["/ignored-path/*"]},
```

Reply:
Ah, my bad, the f string was left over from something I was trying.